trunner/testcase.py

```diff
@@ -308,13 +313,18 @@ class TestCaseFactory:
     def create(test):
         status = TestCase.SKIPPED if test['ignore'] else TestCase.FAILED
         use_sysexec = test['target'] in SYSEXEC_TARGETS
+        if test['target'] in ASH_TARGETS:
+            prompt = ASH_PROMPT
+        else:
+            prompt = DEFAULT_PROMPT
```

Comment:
I don't know if it's a good way to add target dependencies here. I noticed that when a new target comes then the list of arguments to the `TestCaseUnit` also grows. Maybe we should create `TestCaseArgs` or `TestCaseConfig` and make this ifology in the constructor? Then we could pass one argument to `TestCaseUnit` and use it there (maybe unpacking it or using property feature). Then we could also eliminate some ifs from `TestCaseUnit` methods.

Reply:
It's done in the last force push